src/components/PlantsV2/index.tsx

```diff
@@ -39,6 +40,12 @@ export default function PlantsDashboardV2(): JSX.Element {
   const [plantsDashboardPreferences, setPlantsDashboardPreferences] = useState<Record<string, unknown>>();
   const locale = useLocalization();
   const history = useHistory();
+  const numberFormatter = useNumberFormatter();
+  const user = useUser().user;
+  const numericFormatter = useMemo(
+    () => numberFormatter(user?.locale, user?.countryCode),
+    [user?.locale, numberFormatter, user?.countryCode]
+  );
 
```

Comment:
Suggestion:
Move this into a standalone `FormattedNumber` component which takes in the value to render:
```
function FormattedNumber({ value }: Props) {
  const user = useUser().user;
  const numericFormatter = useMemo( ....);

  return numericFormatter.format(value);
}

// in the clients just call
<Typography...>
  <FormatterNumber value={value} />
</Typography>
```

simplifies the code, don't need to pass all those props around
i added links to reference code in the jira earlier in case it helps